gpkit/tests/t_constraints.py

```diff
@@ -199,12 +199,12 @@ def test_posyconstr_in_sp(self):
         y = Variable('y')
         with SignomialsEnabled():
             sig_constraint = (x + y >= 0.1)
-        m = Model(x, [Tight([x >= y], raiseerror=True),
-                      x >= 2, y >= 1, sig_constraint])
+        m = Model(x*y, [Tight([x >= y], raiseerror=True),
+                        x >= 2, y >= 1, sig_constraint])
```

Comment:
The previous cost wasn't necessarily not tight; y could have the value of 2 without any issue.